CHANGELOG.rst

```diff
@@ -24,6 +24,7 @@ Removed
 Fixed
 -----
 - validation no longer throws an error during interactive learning
+- add new parser to handle intent payload input messages
```

Comment:
You did not add a new parser. You updated the server endpoint `/model/parse` to handle also messages with the intent prefix.

Reply:
Thanks, in my mind this code is much related to RASA-X Interactive Learning. This description is better.